packages/schemas/extract/src/merge-request-notes.ts

```diff
@@ -0,0 +1,22 @@
+import type { InferModel } from 'drizzle-orm';
+import { sql } from 'drizzle-orm';
+import { sqliteTable, text, integer, uniqueIndex } from 'drizzle-orm/sqlite-core';
+
+export const mergeRequestNotes = sqliteTable('merge_request_notes', {
+  id: integer('id').primaryKey(),
+  externalId: integer('external_id').notNull(),
+  mergeRequestId: integer('merge_request_id').notNull(), // internalId not mergeRequestId
+  createdAt: integer('created_at', { mode: 'timestamp_ms' }).notNull(),
+  updatedAt: integer('updated_at', { mode: 'timestamp_ms' }).notNull(),
+  authorUsername: integer('author_username').notNull(),
+  authorExternalId: integer('author_external_id').notNull(),
+  body: text('body').notNull(),
```

Comment:
Do we need to store body of the message?

Seems rather unnecessary.

Reply:
gitlab generates automatic messages which we can parse to determine nature of message, although this can be done before storing in the database. I can drop isSystem message as well then if we are only interested in user comments